ext/libp2p/peer.rs

```diff
@@ -422,56 +468,62 @@ mod tests {
         init();
         let cancellation_token = CancellationToken::new();
 
-        let server_id_keys = identity::Keypair::generate_ed25519();
-        let server_peer_id = server_id_keys.public().to_peer_id();
-        let mut server_transport = create_transport(&server_id_keys).unwrap();
+        let listener_id_keys = identity::Keypair::generate_ed25519();
```

Comment:
Note that the new `libp2p-swarm-test` crate can help with all this boilerplate:

https://github.com/libp2p/rust-libp2p/tree/master/swarm-test

Reply:
@mxinden thank you for the tip, the helpers in the swarm-test crate look really helpful! I would love to use [`listen`](https://github.com/libp2p/rust-libp2p/blob/32598d89d6a3008e416679ca4c2705d8b439c541/swarm-test/src/lib.rs#L77), see https://github.com/filecoin-station/zinnia/pull/85#discussion_r1123115694

There is a catch though: the swarm-test crate is not re-exported by libp2p and it's not published as a standalone crate to crates.io either :(

Reply:
Do you know how I can use `swarm-test` from my code?